cmd/fuzz.go

```diff
@@ -18,6 +19,26 @@ var fuzzCmd = &cobra.Command{
 	Long:  `Starts a fuzzing campaign`,
 	Args:  cmdValidateFuzzArgs,
 	RunE:  cmdRunFuzz,
+	// Run dynamic completion of nouns
+	ValidArgsFunction: func(cmd *cobra.Command, args []string, toComplete string) ([]string, cobra.ShellCompDirective) {
```

Comment:
The last concern I had was around the `--help` flag? Like once you add any flag to the command maybe the `--help` flag shouldn't show in the `unusedFlags` object? idk this is minor but wanted to get your guys thoughts